solutions/USACO Feb19-SleepyCowHerding.mdx

```diff
@@ -0,0 +1,59 @@
+---
+id: usaco-915
```

Comment:
change the name of the file so that it's the same as the id; in this case, usaco-915.mdx

Reply:
@thecodingwizard sorry about this, this is the first time I'm hearing about this new naming convention